src/third-party/sleigh/native/sleigh_interface.cpp

```diff
@@ -296,6 +296,9 @@ class TranslationContext
 
     TranslationContext(maat::Arch::Type a, const std::string& slafile, const std::string& pspecfile): arch(a)
     {
+        AttributeId::initialize();
+        ElementId::initialize();
```

Comment:
This is required with 10.2.

We had to make similar changes here https://github.com/lifting-bits/sleigh/commit/f38eee41a289ac5fbf27c6951a3c65e8a48e7bac#diff-aed2f30b604fcd3c832b0255d6cd0ef5a7cca4cab0d8d2c77ee6d125d2fcf301